<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Calculator&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>39</y>
      <width>561</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetFixedSize</enum>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QLineEdit" name="entrada_numero_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>20</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_convertidor_4">
           <property name="text">
            <string>convertir</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>150</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>260</width>
             <height>119</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Box</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::AutoText</enum>
           </property>
           <property name="pixmap">
            <pixmap>C:\Users\SEBASTIAN\OneDrive\Documentos\GitHub\Tercer_Semestre_U\ejercicios_expo\Escudo_udenar.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="pantalla">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>130</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>100</number>
       </property>
       <property name="bottomMargin">
        <number>100</number>
       </property>
       <item>
        <widget class="QRadioButton" name="Decimal_4">
         <property name="text">
          <string>Decimal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Binario_4">
         <property name="text">
          <string>Binario</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Octal_4">
         <property name="text">
          <string>Octal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Hexadecimal_4">
         <property name="text">
          <string>Hexadecimal</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
